CameraOverlay/overlay_all_stats.py

```diff
@@ -129,7 +131,18 @@ def on_message(self, client, userdata, msg):
 				self.data["reed_velocity"] = 0
 				self.data["reed_distance"] = 0
 				self.data["count"] = 0
-
+		
+		elif topic == str(topics.DAShboard.recieve_message):
+			message = msg.payload.decode("utf-8")
+
+			# reset counter once message is recieved 
+			self.frame_counter = 0  
+
+			# Display Message
+			self.message_canvas.draw_text("{0}".format(message), (190, self.text_height * 5), size= 1, colour=Colour.red)
```

Comment:
`"{0}".format(message)` is unnecessary as you're not really doing any formatting to the string. `message` would suffice.